src/views/newGameView.vue

```diff
@@ -12,23 +13,81 @@ const code = codeStore();
 const timer = timerStore();
 const sound = soundStore();
 
-const { setMisses, setGameFalse } = user;
+const { setMisses, setGameFalse, setGameTrue } = user;
 const { startTimer, stopTimer } = timer;
 const { moveIndex, startGame, setMissCount, changeLine } = code;
-const { onCountDown, onFinish } = sound;
+const { onCountDown, onFinish, setSoundCount } = sound;
 
 const upper = ref<HTMLElement>();
 let showMyCodeDialog = ref<boolean>(false);
-
+const missMoves = (eventKey: KeyboardEvent): void => {
+  console.log("misses");
+  setMissCount();
+  setMisses(eventKey);
+  user.setScore();
+  sound.onMiss();
+};
+const finishMove = (event: KeyboardEvent): void => {
+  onFinish();
+  stopTimer();
+  setGameFalse();
+  setSoundCount();
+  finishResetKeyBoardColor(event);
+  openDialog();
+  return;
+};
 const openDialog = (): void => {
   showMyCodeDialog.value = true;
 };
 const closeDialog = (): void => {
   showMyCodeDialog.value = false;
 };
+onBeforeRouteLeave((to, from) => {
+  stopTimer();
+  setGameFalse();
+  setSoundCount();
+  finishResetKeyBoardColor(event);
+});
```

Comment:
ゲーム自体を無効化にしたいので下のコード
それと、リスタートでスコア継続されてしまう！
```suggestion
onBeforeRouteLeave((to, from) => {
  setGameFalse();
  setSoundCount();
  code.resetCode();
  timer.resetTimer();
  resetMisses();
});
```

Reply:
[fb7abe9](https://github.com/recursion-team-o/typing-game/commit/fb7abe96d58ff4b7bba32901d58e2c3e6336935b)